testing/widgets/dropdownbox/MultipleSelection.test.js

```diff
@@ -0,0 +1,32 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+
+fixture('DropDownBox.MultipleSelection')
+  .page('http://localhost:8080/')
+  .beforeEach(async (t) => {
+    await t.resizeWindow(900, 600);
+  });
+
+runManualTest('DropDownBox', 'MultipleSelection', ['jQuery', 'React', 'Vue', 'Angular'], (test) => {
+  test('MultipleSelection', async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await t.click($('.dx-dropdownbox').nth(0))
+      .wait(200);
+
+    await takeScreenshot('dropdownbox_multiple_selection_treebox');
```

Comment:
It is not necessary to add underscores between words, we may use spaces

Reply:
I think it's still a good idea to use underscores here because previous etalons use this format.